website/content/options/rnval/_index.md

```diff
@@ -0,0 +1,20 @@
+```
+usage: rnval [-p] [-m MIN] [-M MAX] [-r RISK] [-h]
+```
+
+calculates the expcected value of a given option by multiplying
+the payoff at each predicted stock price by the probability of
+that stock price happening. This expected profit is divided by
+one over the risk free rate to determine the value of those
+future dollars today. The expected payoff for each strike price
+is then compared with the last traded price to determine if there
+are differences in the market.
```

Comment:
Can you go a little further into an example calculation and what the user should take away from using the function?